client/playbook_runs.go

```diff
@@ -160,6 +160,21 @@ func (s *PlaybookRunService) UpdateStatus(ctx context.Context, playbookRunID str
 	return nil
 }
 
+func (s *PlaybookRunService) RequestUpdate(ctx context.Context, playbookRunID, userID string) error {
```

Comment:
The userID is not needed here, right?
```suggestion
func (s *PlaybookRunService) RequestUpdate(ctx context.Context, playbookRunID) error {
```

Reply:
I thought this would have been caught by the linter :thinking: